config/jobs/kubernetes/sig-release/release-branch-jobs/1.20.yaml

```diff
@@ -361,6 +360,7 @@ periodics:
       - --test-cmd-args=--provider=gce
       - --test-cmd-args=--report-dir=$(ARTIFACTS)
       - --test-cmd-args=--testconfig=testing/load/config.yaml
+      - --test-cmd-args=--testconfig=testing/huge-service/config.yaml
```

Comment:
Have you backported this file to 1.20 branch in perf-tests repo?

Anyway, I don't think we want to change tests in OSS other than in master branch

Reply:
Thanks! This is indeed a good idea. Reverted all the non-master changes. Fixed.

Reply:
Good idea. Removed all the non-master changes. Fixed. :)